### ISSUE TITLE
Trained CASIA to 98.92/93.3 without using center loss

### ISSUE THREAD

**Issue by JianbangZ** [davidsandberg/facenet#473]
When I applied center loss, the final cross entropy is low (~0.5), but the accuracy/tar combination is not as good as just training with softmax. When only training with softmax, the final cross entropy is higher (0.8). This means when introducing center loss to incepiton_resnet_v1 with CASIA dataset, there is actually some overfitting.
